db/fixtures/miq_product_features.yml

```diff
@@ -1837,6 +1837,15 @@
       :description: View Policy Profiles
       :feature_type: view
       :identifier: miq_policy_set_view
+      :children:
+      - :name: List
+        :description: Display All Policy Profiles
+        :feature_type: view
+        :identifier: miq_policy_set_show_list
+      - :name: Show
+        :description: Show Policy Profile
+        :feature_type: view
+        :identifier: miq_policy_set_show
```

Comment:
There's a `miq_policy_set_view` feature,  how is that different from this one?

Reply:
`xx_view` is the parent/folder node to give access to any view related features such as show_list, show or more in some cases, that is consistent with other non-explorer screen product feature structure

Reply:
:+1: